skore/src/skore/sklearn/_estimator/data_accessor.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+from sklearn.utils.fixes import parse_version
+from skrub import _dataframe as sbd
+from skrub import _join_utils
+from skrub._dataframe._common import dispatch
+
+from skore.externals._pandas_accessors import DirNamesMixin
+from skore.sklearn._base import _BaseAccessor
+from skore.sklearn._estimator.report import EstimatorReport
+from skore.sklearn._plot import TableReportDisplay
+
+pandas_version = parse_version(parse_version(pd.__version__).base_version)
+
+
+# TODO: remove when skrub 0.7.0 is released
```

Comment:
I'm thinking that we should do a `_skrub_compat` file for those types of thing. It will come back time to time.